docs/csharp/language-reference/keywords/default.md

```diff
@@ -18,15 +18,13 @@ ms.author: "wiwagn"
 
 # default (C# Reference)
 
-The `default` keyword.
-
 The `default` keyword can be used in the `switch` statement or in a default value expression:
 
 - [The switch statement](switch.md): Specifies the default label.
 
-- [Default value expressions](../../programming-guide/statements-expressions-operators/default-value-expressions.md): Produces the default value of the type. This will be null for reference types and zero for value types, and the 0 bit pattern for structs.
+- [Default value expressions](../../programming-guide/statements-expressions-operators/default-value-expressions.md): Returns the default value of a type: `null` for reference types, zero for numeric types, and zero/`null` filled in members for struct types.
```

Comment:
I'd like to keep the term "Produces" instead of "Returns". Yes, this is a bit pedantic, but expressions don't "return", they are evaluated. Evaluation of an expression produces a result.

Another question: With the new wording, do you think nullable types need to be called out? They weren't earlier, and it both versions, the behavior for `default(int?)` is hinted at but not stated.

Reply:
@BillWagner as for "Produces", I see your point and agree. Will update it and thank you for the clarification.

As for the explanation text, I've thought about removing it at all and having just *Produces the default value of a type*. Thus, inviting a reader to follow the link (I'll make another PR to update the actual topic that describes default value expressions) that provides more details. As for now, there are three places in docs where we explain default type values. What do you think of it?

Reply:
And question: is it possible that folks think of `int?` as a numeric type?

Reply:
> As for now, there are three places in docs where we explain default type values. What do you think of it?

Yes. I like that idea.

>is it possible that folks think of int? as a numeric type? If yes, then they might misread the text. 

It is. That's the best case for calling it out explicitly.

Reply:
@BillWagner I've updated both pages. Please check, if we understand each other correctly. The [default value expression](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/default-value-expressions) page -> another PR.